adhocteam/Head-Start-TTADP
# CD: Run database migrations

**Description of change**
Run database migrations as part of cd pipeline

**_Note for reviewers:_** I'm kind of breaking our pattern here by making the _plain_ yarn command `yarn db:migrate` into the local command, and appending a modifier to the production command (`yarn db:migrate:prod`). I thought this was slightly more fitting because I'm using the non-default .sequelizerc file in production (`.production.sequelizerc`).  I don't really have strong feelings about this. If you want me to flip this around so the migration command for poduction doesn't have a modifier, or if you want me to change the name of one or both .sequalizerc files I'm happy to do that.

**How to test**
- To verify migrations were run against sandbox db, follow new directions for "Interacting with a deployed database" in README.md. Then run `\d` psql command to verify that tables exist.
- After merge to main branch, you can use new directions for "Interacting with a deployed database" to verify migrations were run against the dev db. You'll just have to update the example commands to target the dev space and dev database.  You should also login to cloud.gov and check the log stream for the "tta-smarthub-dev" application. The output from `yarn db:migrate:prod` will be viewable there. This is also how you'll check for migration errors.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/202

**Checklist**

- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
